app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -106,7 +107,8 @@ class AlertFacilityChangeSheet : BaseBottomSheet<
     if (isFacilitySwitchedPreference.get().not()) {
       view.postDelayed(::closeSheetWithContinuation, 100)
     } else {
-      requireDialog().window!!.setDimAmount(0.32F)
+      val backgroundDimAmount = requireContext().resolveFloat(android.R.attr.backgroundDimAmount)
+      requireDialog().window!!.setDimAmount(backgroundDimAmount)
```

Comment:
Which extension model are we talking about here?

Reply:
In line 100 we are hardcoding the dim amount to 0F should we not be using the method requireContext().resolveFloat() to set the dim amount there as well?

Reply:
Ah, I see what you mean. It's not necessary there since we just want to set it to say "Don't dim the background" which is just setting it to 0F. When we set it to the an actual dim though, we want to load it from the resources because otherwise we have to hardcode the value to use in the code.

Reply:
Got it